style-dictionary2/build-theme.js

```diff
@@ -3,7 +3,7 @@ function build(source, destination) {
     source,
     platforms: {
       js: {
-        transformGroup: 'js',
+        transforms: ['attribute/cti', 'name/cti/kebab', 'size/rem', 'color/hex'],
```

Comment:
I used the same transforms from the `js` group, but replaced the `name/cti/pascal` with `name/cti/kebab` to get us that other naming format (see https://amzn.github.io/style-dictionary/#/transform_groups?id=js)

Reply:
dang, build does not like kebab
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/114949086/197068433-ac6fe2d4-6f89-4005-a09d-15f7dff8f3d0.png">